extension.js

```diff
@@ -128,6 +129,11 @@ class ReferenceProvider {
         if (!newUriPath.match(/\.js$/i)) newUriPath += '.js';
 
         const newUri = vscode.Uri.file(newUriPath);
+
+        if (!fs.existsSync(newUri.fsPath)) {
```

Comment:
This was a previously suggested solution. Instead of slowing down response with a sync call, it would be better to find a faster solution.

In fact we are using vscode API properly and handle the rejection. But vscode still throws a `[uncaught exception]: [object Object]`.

Maybe we should report this like [this](https://github.com/Microsoft/vscode/issues/31319).

A quick fix will be a try-catch block IMO.